samples/boards/stm32/power_mgmt/serial_wakeup/sample.yaml

```diff
@@ -15,3 +15,4 @@ tests:
       - nucleo_wb55rg
     filter: dt_compat_enabled("zephyr,power-state")
     extra_args: "CONFIG_DEBUG=y"
+    platform_allow: nucleo_wb55rg stm32l562e_dk
```

Comment:
why don't you filter using stm32 uart compatible instead?

Reply:
Agreed.
This test will only work on the boards that are provided with an overlay to set up a clock configuration compatible with stop modes.